README.md

```diff
@@ -367,27 +297,28 @@ chocolateyfeature {'viruscheck':
 
 Requires 0.9.10+ for this feature.
 
-Use FIPS Compliant Checksums - Ensure checksumming done by choco uses
-FIPS compliant algorithms. Not recommended unless required by FIPS Mode.
+**Use FIPS Compliant Checksums** - Ensures checksumming done by Chocolatey uses
```

Comment:
`choco` is the name of the binary. Chocolatey is the official name. They are interchangeable. If you prefer using the proper name though 👍